pydrive2/auth.py

```diff
@@ -196,21 +130,96 @@ def __init__(
         self.settings = settings or self.DEFAULT_SETTINGS
         ValidateSettings(self.settings)
 
-        storages, default = self._InitializeStoragesFromSettings()
-        self._storages = storages
-        self._default_storage = default
+        self._service = None
+        self._client_config = None
+        self._oauth_type = None
+        self._flow = None
+        self._storage_registry = {}
+        self._default_storage = None
+        self._credentials = None
+
+    # Lazy loading, read-only properties
+    # these look like functions but are actually getters for the read only properties
```

Comment:
I don't think we need this explanation, it's quite common pattern

Reply:
almost all of these properties should be private? which ones do we want to expose?